src/panels/lovelace/components/hui-action-editor.ts

```diff
@@ -121,6 +142,11 @@ export class HuiActionEditor extends LitElement {
             ${this.hass!.localize(
               "ui.panel.lovelace.editor.action-editor.actions.default_action"
             )}
+            ${this.defaultAction
+              ? ` (${this.hass!.localize(
+                  `ui.panel.lovelace.editor.action-editor.actions.${this.defaultAction}`
+                ).toLowerCase()})`
```

Comment:
Should we lowercase this? Shouldn't the translation handle that?

Reply:
We can lowercase the translation but we will have to capitalize the first letter when using as a select option 😅